itk_siteimprove.module

```diff
@@ -9,6 +9,7 @@
  */
 function itk_siteimprove_preprocess_page(&$variables) {
   $settings = \Drupal::getContainer()->get('itk_siteimprove.settings');
+  $use_cookieinformation = \Drupal::config('itk_siteimprove.config')->get('use_cookieinformation');
```

Comment:
Please use CamelCase in your variable names for consistency with the rest of the variable names in the file. I don't understand why this variable is called "use_cookieinformation", the word use implies the variable is a boolean, but later it is checked as a number. I am guessing Drupal is returning a number, in my opinion this should be cast to a boolean before you use it later on.

Reply:
Fixed